kolibri/plugins/learn/assets/src/views/content-card/index.vue

```diff
@@ -62,6 +77,16 @@
         default: false,
       },
     },
+    data() {
+      return {
+        hasTooltip: false,
+      };
+    },
+    updated() {
+      this.$nextTick(() => {
+        this.hasTooltip = this.$el.querySelector('h3 .js-shave') !== null;
```

Comment:
This selector is a bit brittle - it will break if we change the template DOM structure.

Instead you can add a `ref` to the `h3` element and then reference it directly:

```javascript
 this.hasTooltip = Boolean(this.$refs.title.querySelector('.js-shave'));
```

Reply:
Cool! I didn't know you can do something like this